tool/tsh/tsh.go

```diff
@@ -796,9 +809,41 @@ func Run(args []string, opts ...cliOption) error {
 	return trace.Wrap(err)
 }
 
+// onVersion prints version info.
+func onVersion(cf *CLIConf) error {
+	format := strings.ToLower(cf.Format)
+	switch format {
+	case teleport.Text:
```

Comment:
In all these switches, would it make sense to add empty string (`case teleport.Text, ""`) to be on the safe side and default to text format if the specified format is empty for some reason?